src/components/TopBar/CallTime.vue

```diff
@@ -204,7 +236,10 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-
+.one-hour-hint{
+	display : flex ;
+	padding : 6px;
```

Comment:
Code style here:
- No space before ":"
- No space before ";"

Reply:
And padding should be set based on multiples of --default-grid-baseline.

Also keep in mind sides might need more than top and bottom to look visually consistent. :)